koiki/tests/test_models.py

```diff
@@ -0,0 +1,91 @@
+from unittest import TestCase
+
+from koiki.models import Shipment, Sender, Recipient
+from koiki.woocommerce.models import Vendor, Billing, Shipping
```

Comment:
it's getting more clear as we go but I'm still not sure where we should move this module. `woocommerce` or `api.woocommerce`? I'll leave it for later because we might have more insights.

Reply:
I've been checking it and I think we will eventually extract the logic that turns the parsed dictionaries into Woocommerce models onto something that gets called from the view. This will leave with two things: 1) Convert from dicts to models 2) call Koiki's module to send those models.

Reply:
I agree. Since separting the vendor's from Woocomerce is not something directly related to koiki's code but that we could use generically for different shipping API's in long term could be better to separate woocommerce from koiki.